### ISSUE TITLE
Doesn't support latest version of firebase-admin@11.0.0

### ISSUE THREAD

**Issue by xmany** [firebase/firebase-functions#1157]
After updating firebase-admin to version 11.0.0, `npm install` would return:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: functions@2.0.0
npm ERR! Found: firebase-admin@11.0.0
npm ERR! node_modules/firebase-admin
npm ERR!   firebase-admin@"^11.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer firebase-admin@"^8.0.0 || ^9.0.0 || ^10.0.0" from firebase-functions@3.21.2
npm ERR! node_modules/firebase-functions
npm ERR!   firebase-functions@"^3.21.2" from the root project
```

Downgrade `firebase-admin` to 10.3.0, it works again.

**Comment by Burgov** [comment]
It's fixed, but yet to be released: https://github.com/firebase/firebase-functions/pull/1152/commits/211d7108000a3f90d2f613e46846842ca5a5fafd

**Comment by Burgov** [comment]
ah wait it was released, in v3.22.0

**Comment by xmany** [comment]
> ~It's fixed, but yet to be released: [211d710](https://github.com/firebase/firebase-functions/commit/211d7108000a3f90d2f613e46846842ca5a5fafd)~
> 
> It was fixed in [v3.22.0](https://github.com/firebase/firebase-functions/blob/v3.22.0/package.json#L232)

Thanks!
